textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcher.java

```diff
@@ -57,19 +53,11 @@
 	private java.util.regex.Pattern javaPattern;
 	
 	
-    public RegexMatcher(IPredicate predicate) throws DataFlowException{
-    	this (predicate, true);
-    }
-
-    public RegexMatcher(IPredicate predicate, boolean useTranslator) throws DataFlowException{
```

Comment:
Translator will always be used to attempt to translate regex into an lucene query.